index.html

```diff
@@ -133,80 +212,163 @@ <h2 class="project-title">Multi-Post Stories</h2>
+            </div>
+            <button class="card-button" type="button">See Project</button>
           </div>
         </div>
       </div>
     </section>
+    <!-- desktop -->
     <section>
-      <div class="my-individual-project">
+      <div class="my-individual-project desktop">
         <div class="content-wrapper">
-          <div class="project-snaphot">
-            <img src="./assets/orange-project.png" alt="porfolio-snapshot" />
+          <div class="project-snapshot even">
+            <img
+              src="./assets/Snapshoot Portfolio.png"
+              alt="porfolio-snapshot"
+            />
           </div>
           <div class="project-details">
-            <h2 class="project-title">Tonic</h2>
+            <div class="project-description even">
+              <h2 class="project-title">Facebook 360</h2>
+              <div class="sub-title">
+                <span>CANOPY</span>
+                <img src="./assets/Counter.png" alt="counter" />
+                <span class="dev-title">Back End Dev</span>
+                <img src="./assets/Counter.png" alt="counter" />
+                <span class="year-produced">2021</span>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="stacks">
+                <li>HTML</li>
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button class="card-button active" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- mobile -->
+    <section>
+      <div class="my-individual-project mobile">
+        <div class="content-wrapper">
+          <div class="project-snapshot even">
+            <img
+              src="./assets/orange-project.png"
+              alt="porfolio-snapshot"
+            />
+          </div>
+          <div class="project-details">
+            <div class="project-description even">
+              <h2 class="project-title">Tonic</h2>
+              <div class="sub-title">
+                <span>CANOPY</span>
+                <img src="./assets/Counter.png" alt="counter" />
+                <span class="dev-title">Back End Dev</span>
+                <img src="./assets/Counter.png" alt="counter" />
+                <span class="year-produced">2021</span>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="stacks">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button  class="card-button" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] At midpoint 768px which is the last view point for the mobile version. Your design is distorted, and project lists section is absent. Kindly check below screen record of how your design looks like at this moment and fix where the error is.

### App Behaviour
[your-app.webm](https://github.com/CobbyElsonfx/Portfolio-Website/assets/26597432/8500d3a3-e473-4734-8c44-a4242097f18e)

Reply:
@Qoosim  can you please checkout it again.